### PR TITLE
Add some trust_return_value_nullability's for gtk

### PR DESCRIPTION
**Pull request by sophie-h**

### CODE REVIEW

**Comment by sdroege** [comment]
@sophie-h You might want to create an issue like https://github.com/gtk-rs/gtk-rs/issues/163 also here in gtk4-rs to keep track of this :)

File: gtk4/Gir.toml [hunk 1]
```diff
@@ -404,33 +404,48 @@ status = "generate"
...
 generate_builder = true
+# upstream_needs_fix
+trust_return_value_nullability = true
     [[object.signal]]
     name = "activate-link"
     inhibit = true
+    [[object.function]]
+    name = "get_program_name"
+        [object.function.return]
+        nullable = true
```
Comment (sdroege) [inline gtk4/Gir.toml:415]: Please link to an upstream issue/commit for these things, or mention an upstream version where it's fixed. That way we can properly keep track of it and remove the override when possible.

**Review by sdroege — COMMENTED** [review]
Looks good to me otherwise, thanks for taking care of this :)

File: gtk4/Gir.toml (hunk 1 above)
Comment (sophie-h) [reply gtk4/Gir.toml:416]: Should me much clearer now.

File: generator.py [hunk 2]
```diff
@@ -171,7 +171,7 @@ def main():
         conf["gir_path"] = "./gir/target/release/gir"
 
     print('=> Regenerating crates...')
-    if not regen_crates(".", conf):
+    if not regen_crates("gtk4", conf):
```
Comment (bilelmoussaoui) [inline generator.py:174]: I don't think this change is needed?

Comment (sophie-h) [reply generator.py:174]: Oh wow, no! that was just for speeding things up. Will add another MR to make this an option.

**Review by sdroege — APPROVED** [review]

**Comment by bilelmoussaoui** [comment]
Thanks!

**Pull request merged**